Switch to a branch packaged in Fedora, which currently has a compatible API.

--- a/src/arch/Cargo.toml
+++ b/src/arch/Cargo.toml
@@ -9,7 +9,7 @@
 bitflags = "1.3.2"
 derive_more = { version = "0.99.17", default-features = false, features = ["from"] }
 kvm-bindings = { version = "0.6.0", features = ["fam-wrappers"] }
-kvm-ioctls = "0.12.0"
+kvm-ioctls = "0.13.0"
 libc = "0.2.117"
 linux-loader = "0.8.1"
 versionize = "0.1.6"
--- a/src/cpuid/Cargo.toml
+++ b/src/cpuid/Cargo.toml
@@ -7,7 +7,7 @@
 
 [dependencies]
 kvm-bindings = { version = "0.6.0", features = ["fam-wrappers"] }
-kvm-ioctls = "0.12.0"
+kvm-ioctls = "0.13.0"
 thiserror = "1.0.32"
 
 arch = { path = "../arch" }
--- a/src/vmm/Cargo.toml
+++ b/src/vmm/Cargo.toml
@@ -9,7 +9,7 @@
 derive_more = { version = "0.99.17", default-features = false, features = ["from"] }
 event-manager = "0.3.0"
 kvm-bindings = { version = "0.6.0", features = ["fam-wrappers"] }
-kvm-ioctls = "0.12.0"
+kvm-ioctls = "0.13.0"
 lazy_static = "1.4.0"
 libc = "0.2.117"
 linux-loader = "0.8.1"
